service/src/main/java/bio/terra/workspace/common/utils/ControllerValidationUtils.java

```diff
@@ -166,6 +166,21 @@ public static void validateSasExpirationDuration(
     }
   }
 
+  /**
+   * Validate an azure blob name. Blob names must be > 1 char and < 1024 chars in length.
+   *
+   * @param blobName Blob name to validate
+   */
+  public static void validateSasBlobName(@Nullable String blobName) {
```

Comment:
It would be good to add unit test coverage of this in service/src/test/java/bio/terra/workspace/common/utils/ControllerValidationUtilsTest.java.